e2/src/main/index.ts

```diff
@@ -126,11 +128,15 @@ async function whenReady() {
   })
 
   try {
-    await autoUpdater.checkForUpdatesAndNotify();
+    update = await autoUpdater.checkForUpdatesAndNotify();
     setInterval(async () => {
       try {
+        if (update?.updateInfo.version) {
```

Comment:
```suggestion
        if (update?.isUpdateAvailable) {
```

Why not use `isUpdateAvailable` ? Is there still a problem on Linux?